src/locales/en.json

```diff
@@ -240,6 +240,10 @@
     "whyWeDoItHeader": "Why we do it",
     "whyWeDoItText": "We have a moral obligation to preserve nature and its features, because protecting the environment protects humanity from toxic substances and our quality of food. We believe that the new generation in MENA should be supported with the right tools to believe and act for the benefit of humanitarian and social issues, on the local and global scale. Given the right tools, it is not impossible to raise a generation of activists."
   },
+  "loginRegister": {
+    "anEmailSent": "An email message has been sent to ",
```

Comment:
the turkish and en translations are wrong.

google string interpolation i18n

Reply:
Added interpolation. Translation accuracy will be tested tomorrow. PTAL.

Reply:
sorry i didnt mean the translation itself is wrong but wrong in the sense of not being interpolated